vumi_http_proxy/test/test_clickme.py

```diff
@@ -12,12 +12,25 @@ def test_click(self):
         self.initializers = []
         self.patch(http_proxy.Initialize, 'main',
                    lambda x: self.initializers.append(x))
-        result = runner.invoke(clickme.cli)
-        [initializer] = self.initializers
-        self.assertEqual(result.exit_code, 0)
+        fake_blacklist = ["foo", "bar"]
+        filename = self.make_blacklist(fake_blacklist)
+        result = runner.invoke(clickme.cli, ['--blacklist', filename])
+        self.assertEqual(result.exception, None)
         self.assertEqual(result.output.splitlines(), [
            'Starting connection to 0.0.0.0:8080',
         ])
+        self.assertEqual(result.exit_code, 0)
+
+        [initializer] = self.initializers
         self.assertEquals(type(initializer), http_proxy.Initialize)
         self.assertEquals(initializer.port, 8080)
         self.assertEquals(initializer.ip, "0.0.0.0")
+        self.assertEquals(initializer.blacklist, ["foo", "bar"])
+
+    def make_blacklist(self, blacklist):
+        filename = self.mktemp()
+        with open(filename, 'w') as stream:
+            stream.write("proxy-blacklist:\n")
+            for i in blacklist:
+                stream.write(" - "+i+"\n")
```

Comment:
PEP8 will probably complain that there aren't spaces around the `+` symbols in `+i+`. `i` is also perhaps not the best variable name for a variable that references an IP address?